pkg/pxc/users/users.go

```diff
@@ -0,0 +1,73 @@
+package users
+
+import (
+	"database/sql"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/pkg/errors"
+)
+
+type Manager struct {
+	db *sql.DB
+}
+
+type SysUser struct {
+	Name  string   `yaml:"username"`
+	Pass  string   `yaml:"password"`
+	Hosts []string `yaml:"hosts"`
+}
+
+func NewManager(hosts []string, rootPass string) (Manager, error) {
```

Comment:
Why are the multiple hosts allowed? As I see just service name used to connect to PXC.

Reply:
removed https://github.com/percona/percona-xtradb-cluster-operator/pull/418/commits/9a770955143b4e0e1b00e53653fdb45ab333357e